lib/createExplorer.js

```diff
@@ -47,9 +47,6 @@ module.exports = function (options) {
         const directory = (searchPathIsDirectory)
           ? absoluteSearchPath
           : path.dirname(absoluteSearchPath);
-        if (directoryCache && directoryCache.has(directory)) {
-          return directoryCache.get(directory);
-        }
```

Comment:
Ok ... so this doesn't relate to the bug, but is just code cleanup — is that right?